app/views/miq_request/_request.html.haml

```diff
@@ -112,7 +112,7 @@
             = button_tag(t, :class => "btn btn-primary disabled")
 - else
   = request_details_summary(@miq_request, @user)
-- if @miq_request.request_logs.any?
+- if @miq_request.request_logs.any? && role_allows?(:feature => 'miq_request_show_logs')
```

Comment:
I don't know which is more resource intensive but logically, I think we should do the feature check first... so...

```ruby
if role_allows?(:feature => 'miq_request_show_logs') && @miq_request.request_logs.any? 
```

Reply:
changed